.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
```

Comment:
Can we have the rationale and expected outcomes of the template documented somewhere in the repo itself? Perhaps in a top-level ADR folder.

Reply:
Relatedly, if one were to measure the impact of having this template, what would the hypotheses be?  I want to ensure we are aligned on the value of this effort (rollout, adopt, future maintenance).

Reply:
Idea: You can start with an ADR in `edx-platform` for now, since that's where we're rolling this out - and we need a place document this decision. Each repo that chooses to adopt this template, can have their own ADR for the adoption and link to the one in edx-platform. The repo may have their own modifications/additions, which they can capture in their repo-specific ADR.